client/src/components/Page/Page.js

```diff
@@ -11,6 +11,7 @@ const Page = ({ className, children, header, headerClassName }) => (
     </div>
     <footer className="row">
       <img alt="fostive logo" src={logoSrc} />
+      <p>Brought to you with 💛 &nbsp;by <a href="https://github.com/fostive">Fostive</a></p>
```

Comment:
@DianaPerkinsDesign do you have any deep thoughts on opening a link in a new tab vs not? On one hand, I don't want to interrupt what the user is currently doing with the app by loading https://github.com/fostive. On the other hand, I generally get frustrated with sites that open new tabs needlessly.

Reply:
I usually like to leave that up to the user to decide, so I don't typically do that. But it is an external link, so we could totally open it in a new window (as long as we properly warn screenreaders)! 

Do you want me to switch it? 

Sorry my answer isn't more firm 😄

Reply:
New window would be my vote, because it's part of an unrelated flow the user is in, and they may accidentally tap it.